modules/apps/object/object-service/src/main/java/com/liferay/object/internal/security/permission/resource/util/ObjectDefinitionResourcePermissionUtil.java

```diff
@@ -120,4 +139,77 @@ private static String _getPermissionsSupports(
 		return permissionsSupports;
 	}
 
+	private static String _getRootDescendantNodeObjectDefinitionsModelResources(
+			ObjectActionLocalService objectActionLocalService,
+			ObjectDefinitionPersistence objectDefinitionPersistence,
+			ObjectDefinition objectDefinition, TreeFactory treeFactory)
+		throws Exception {
+
+		if (!objectDefinition.isRootNode()) {
+			return StringPool.BLANK;
+		}
+
+		Tree tree = treeFactory.create(
+			objectDefinition.getObjectDefinitionId());
+
+		int index = DEFAULT_WEIGHT;
```

Comment:
@p-albuquerque can you rename this variable to `weight`? Sorry about my misunderstanding.